packages/clerk-js/src/core/clerk.ts

```diff
@@ -660,6 +664,16 @@ export default class Clerk implements ClerkInterface {
     return this.buildUrlWithAuth(this.#environment.displayConfig.organizationProfileUrl);
   }
 
+  public redirectToSatellite = async (): Promise<unknown> => {
+    const redirectUrl = new URL(window.location.href).searchParams.get('redirect_url') as string;
```

Comment:
❓ Do we need to parse the redirect_url parameter as a URL? How will this behave if it's a path and not a full URL? My suggestion is to decode the search param and use it as is.